Mathlib/Topology/Algebra/Module/LinearMap.lean

```diff
@@ -441,6 +441,24 @@ theorem comp_id (f : M₁ →SL[σ₁₂] M₂) : f.comp (id R₁ M₁) = f :=
 theorem id_comp (f : M₁ →SL[σ₁₂] M₂) : (id R₂ M₂).comp f = f :=
   ext fun _x => rfl
 
+section
+
+variable {R E F : Type*} [Semiring R]
+  [TopologicalSpace E] [AddCommMonoid E] [Module R E]
+  [TopologicalSpace F] [AddCommMonoid F] [Module R F]
+
+/-- `g ∘ f = id` as `ContinuousLinearMap`s implies `g ∘ f = id` as functions. -/
+lemma LeftInverse.of_composition {f : E →L[R] F} {g : F →L[R] E}
```

Comment:
```suggestion
lemma leftInverse_of_comp {f : E →L[R] F} {g : F →L[R] E}
```
no need for a dot in the name, as this is not eligible to dot notation.

Reply:
Good point; changed.